pkg/binlog/event/event.go

```diff
@@ -27,6 +27,12 @@ import (
 	"github.com/pingcap/dm/pkg/gtid"
 )
 
+// flags used in Header
+const (
+	HeaderFlagSuppressUse uint16 = 0x0008 // LOG_EVENT_SUPPRESS_USE_F
+	HeaderFlagRelayLog    uint16 = 0x0040 // LOG_EVENT_RELAY_LOG_F
```

Comment:
we can use `LOG_EVENT_SUPPRESS_USE_F` and `LOG_EVENT_RELAY_LOG_F` from go-mysql/replication/const.go directly?